packages/data-service-generator/src/utils/dynamic-installation/DynamicPackageInstallationManager.ts

```diff
@@ -42,6 +47,8 @@ export class DynamicPackageInstallationManager {
 export interface PackageInstallation {
   name: string;
   version: string | null;
+  pluginId: string;
+  settings: { [key: string]: any };
```

Comment:
Let's create a more realistic type where at least we define the properties used withing the methods that accept this type.

```suggestion
  settings: { local: boolean,  destPath: string, [key: string]: any };
```